buildConfig.py

```diff
@@ -0,0 +1,48 @@
+import os
+import threading
+from chamberInfo import chamber_config,GEBtype
+from qcoptions import parser
+
+(options, args) = parser.parse_args()
+
+if os.getenv('DATA_PATH') == None or os.getenv('DATA_PATH') == '':
+  print 'You must source the environment properly, DATA_PATH is not set'
+  exit(0)
+if os.getenv('CONFIG_PATH') == None or os.getenv('CONFIG_PATH') == '':
+  print 'You must source the environment properly, CONFIG_PATH is not set'
+  exit(0)
+if os.getenv('BUILD_HOME') == None or os.getenv('BUILD_HOME') == '':
+  print 'You must source the environment properly, BUILD_HOME is not set'
+  exit(0)
+
+
+def launchScurveScan(link,ztrim,cName,cType):
```

Comment:
This function is being repeated a lot of places (of course with slight changes), would suggest to at the very least rename to something descriptive, an possibly abstract into a wrapper *if* possible